kartothek/io/testing/index.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pd
+import pytest
+
+from kartothek.core.factory import DatasetFactory
+from kartothek.core.index import ExplicitSecondaryIndex
+from kartothek.io.eager import (
+    read_dataset_as_metapartitions,
+    store_dataframes_as_dataset,
+)
+
+
+def index_dct_regulator(indices_dict):
+    """
+    Output
+    `kartothek.core.dataset.DatasetMetadataBase.indices.index_dct`
+    in standard format.
+    """
+    return [(k, sorted(v)) for k, v in sorted(indices_dict.items())]
+
+
+def index_dcts_are_equal(indices1, indices2):
```

Comment:
Use a `assert obj1 == obj2` instead of `assert are_objects_equal(obj1, obj2)` since `assert` may print better error messages